components/ResultsPage/index.tsx

```diff
@@ -74,31 +74,42 @@ const ResultsPage: React.VFC<{
   let listLinks: {
     text: string
     url: string
+    idSuffix: string
```

Comment:
with the suggestion above this is not required

Reply:
@MarcoGoC  An ID suffix is in someway required for listLinks as unlike the Benefits card, there is no benefitKey passed down to them. I did remove them in lieu of the benefits key on the BenefitsCard component.